specification/EnterpriseKnowledgeGraph/data-plane/Microsoft.EnterpriseKnowledgeGraph/stable/2018-12-03/EnterpriseKnowledgeGraphSwagger.json

```diff
@@ -0,0 +1,2001 @@
+        "tags": [
+          "Converse"
+        ],
+        "summary": "Get conversation result based on the natural language utterance.",
+        "operationId": "ExecNLQuery",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "query",
+            "name": "service",
+            "description": "Decide to get result from which service, e.g. AI-V1.0, AI-V2.0, Search or any combination of them. \r\n1=AI-V1.0; \r\n2=AI-V2.0; \r\n3=AI-V1.0 Then AI-V2.0; \r\n4=Search; \r\n5=AI-V1.0 Then Search; \r\n7=Default; \r\n",
+            "type": "integer",
+            "format": "int32",
+            "enum": [
+              1,
+              2,
+              3,
+              4,
+              5,
+              7
+            ]
+          },
+          {
+            "in": "body",
+            "name": "body",
+            "schema": {
+              "$ref": "#/definitions/ConverseRequest"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/ConverseResponse"
+            }
+          },
+          "default": {
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/entitydocument/entities/{id}": {
+      "get": {
+        "tags": [
+          "EntityDocument"
+        ],
+        "summary": "Retrieve an entity document by a given entity id",
+        "operationId": "GetEntityDocument",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "path",
+            "name": "id",
+            "description": "Entity Id",
+            "required": true,
+            "type": "string"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/EntityDocument"
+            }
+          },
+          "default": {
+            "schema": {
+              "$ref": "#/definitions/NotFoundResult"
+            }
+          }
+        }
+      }
+    },
+    "/entitydocument/entities": {
+      "post": {
+        "tags": [
+          "EntityDocument"
+        ],
+        "summary": "Retrieve a dictionary of with entity ids as keys and corresponding entity documents as values",
+        "operationId": "GetEntityDocuments",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "body",
+            "name": "options",
+            "schema": {
+              "$ref": "#/definitions/SearchOptions"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
```

Comment:
Will the number of items/the size of the data ever require the service to do paging?